contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -227,7 +258,43 @@ abstract contract PoSValidatorManager is
         uint32 messageIndex
     ) external {
         // Ignore the return value here to force end validation, regardless of possible missed rewards
-        _initializeEndPoSValidation(validationID, includeUptimeProof, messageIndex);
+        _initializeEndPoSValidation(validationID, includeUptimeProof, messageIndex, address(0));
+    }
+
+    /**
+     * @notice See {IPoSValidatorManager-forceInitializeEndValidation}.
+     */
+    function forceInitializeEndValidation(
+        bytes32 validationID,
+        bool includeUptimeProof,
+        uint32 messageIndex,
+        address rewardRecipient
+    ) external {
+        // Ignore the return value here to force end validation, regardless of possible missed rewards
+        _initializeEndPoSValidation(validationID, includeUptimeProof, messageIndex, rewardRecipient);
+    }
+
+    function changeRewardRecipient(bytes32 validationID, address rewardRecipient) external {
+        PoSValidatorManagerStorage storage $ = _getPoSValidatorManagerStorage();
+
+        if ($._posValidatorInfo[validationID].owner != _msgSender()) {
+            revert UnauthorizedOwner(_msgSender());
+        }
+
+        $._rewardRecipients[validationID] = rewardRecipient;
```

Comment:
With the current implementation, I think we should check that `rewardRecipient != address(0)`, since setting it to `address(0)` has the counterintuitive function of sending the rewards to the owner.